### PR TITLE
Sprint 48

### PR DESCRIPTION
**Pull request by ericsorenson**
# Sprint 48

- [6281](https://github.com/flexion/ef-cms/issues/6281): Health and Configuration Endpoints
- [752](https://trello.com/c/TTodEJ7m/752-date-input-validation): Date input validation
- [761](https://trello.com/c/MT7Lvi95/761-icons-in-docket-record-misaligned): Icons in docket record misaligned
- [768](https://trello.com/c/9aC7cmqV/768-use-of-bold-in-tables): Use of bold in tables
- [636](https://trello.com/c/0cHyKz2D/636-explore-merging-the-document-docketrecord): Merge document and docket record
- [763](https://trello.com/c/jBarzHTL/763-add-null-state-for-when-theres-no-respondent-counsel): Add null state for when there's no respondent counsel
- [769](https://trello.com/c/9GOHXlgH/769-document-view-doc-titles-line-height-is-too-close): Document view title line height too close
- [757](https://trello.com/c/sV4JGGa8/757-spacing-between-labels-and-data): Spacing between labels and data
- [781](https://trello.com/c/NwGuwG4A/781-deadline-date-validation-error): Deadline date validation error

Before deploying, please run `./docker-to-ecr.sh`

### CODE REVIEW

File: docs/TROUBLESHOOTING.md [hunk 1]
```diff
@@ -102,3 +102,21 @@ An error occurred (NotAuthorizedException) when calling the AdminConfirmSignUp o
 curl: (3) URL using bad/illegal format or missing URL```
 
 This happened when duplicate API gateways (i.e., `gateway_api_$ENV`) were created due to a Terraform state sync problem.
+
+### Cognito Authorizer Error
```
Comment (julialeague) [inline docs/TROUBLESHOOTING.md]: I don't think we need to document this here, since we found the root cause of the issue and fixed it in #395.

File: shared/src/business/useCases/health/getHealthCheckInteractor.js [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+  }
+};
+
+const getDynamsoftStatus = async ({ applicationContext }) => {
+  const axios = applicationContext.getHttpClient();
+
+  const source = handleAxiosTimeout(axios);
+
+  try {
+    // Currently, dynamsoft is only deployed on the court's staging environment as it shares a license with
```
Comment (julialeague) [inline shared/src/business/useCases/health/getHealthCheckInteractor.js:55]: Is there an issue in the backlog/Trello to address this?

**Comment by julialeague** [comment]
We use this list when performing a code review to ensure that all tasks have been completed.

- [x] If desired, run `docker system prune` to remove any unused docker images from previous code review.
- [x] Fetch the pull request for the sprint (e.g., `git fetch origin pull/{PR #}/head:sprint-{Sprint #}`), and then switch to that branch (e.g. `git checkout sprint-{Sprint #}`)
- [x] review the pull request itself, to get oriented
	- [x] read the description of the pull request, which should summarize the changes made
	- [x] read through every task on the Scrum board that's encompassed by this pull request
	- [x] read the description of the commits that comprise the pull request
- [x] stand up the site locally, with `./docker-run.sh`
	- [x] test all functionality in all major browsers, emphasizing the functionality that this pull request addresses
		- [x] for internal Court functionality, perform the most thorough testing in Chrome, though also test in Edge and Firefox
		- [x] for public-facing functionality, test in browsers consistent with [public browser use data](https://analytics.usa.gov/)
		- [x] test in Mobile Safari and Mobile Chrome (or an emulator like Chrome DevTools), with the caveat that not all internal Court functionality will be necessary on these platforms
	- [x] use an automated audit tool for code quality and practices (recommended: [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools/), aka [Lighthouse](https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk))
		- [x] look at efficiency of page loads, asset sizes, HTTP connection management, etc.
	- [x] review for accessibility
		- [x] use an automated audit tool, such as Chrome Audit or aXe
		- [x] navigate site only with the keyboard
		- [x] use VoiceOver or Narrator to navigate the site with audio only, with the display turned off
		- [x] manually test anything that pa11y cannot test automatically (e.g., contrast of text over images)
- [x] [review static code analysis results](https://sonarcloud.io/organizations/flexion-github/projects) in the vendor’s SonarCloud environment
- [x] examine OWASP ZAP output in `docs/`, to ensure that any errors are known to be false positives or have been previously declared to be acceptable
- [x] skim all new code, in the context of existing code, [looking for problems](#what-we-look-for) (knowing that the vast majority of new code will be covered by tests)
- [x] review all tests
	- [x] look at code coverage of tests in [the vendor's SonarCloud environment](https://sonarcloud.io/organizations/flexion-github/projects)
	- [x] methodically review all new tests for correctness, quality of naming
- [x] determine what code isn’t tested, review that rigorously
- [x] review documentation to ensure that it matches changes
- [x] provide comments on the pull request on GitHub, as necessary
	- [x] for comments that are specific to a particular line of code, comment on those specific lines
	- [x] for comments that are more general, attach the comment to a random line in `README.md` (as opposed to commenting on the pull request itself), to be able to use GitHub's ability to thread discussions on those comments
- [x] for each feature-level bug (i.e., it’s working as designed, but designed wrong), open a new issue and put it in the backlog

File: docs/TROUBLESHOOTING.md (hunk 1 above)
Comment (ericsorenson) [reply docs/TROUBLESHOOTING.md]: Updated!

File: shared/src/business/useCases/health/getHealthCheckInteractor.js (hunk 2 above)
Comment (julialeague) [reply shared/src/business/useCases/health/getHealthCheckInteractor.js:55]: @mmarcotte is going to look further into whether there's a need for another Dynamsoft account - marking this comment as resolved.

**Review by julialeague — APPROVED** [review]
@mmarcotte, I recommend you accept this PR into the court's staging environment. Thanks!
